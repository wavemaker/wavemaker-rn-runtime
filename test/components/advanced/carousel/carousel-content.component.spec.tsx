import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmCarouselContent from '@wavemaker/app-rn-runtime/components/advanced/carousel/carousel-content/carousel-content.component';

describe('Test CarouselContent component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmCarouselContent name="test_CarouselContent"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});