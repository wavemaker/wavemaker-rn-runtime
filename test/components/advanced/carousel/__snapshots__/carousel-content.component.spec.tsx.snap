// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WmCarouselContent Component should render children components correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <View>
    <Text>
      Carousel Item
    </Text>
  </View>
</View>
`;

exports[`WmCarouselContent Component should render the component without crashing 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "center",
      "width": "100%",
    }
  }
/>
`;
